actions/review_pull.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: review_pull
+runner_type: python-script
+description: Retrieve information about a particular Github issue.
```

Comment:
Looks like metadata update needed here

Reply:
Hi, thanks for your answer, i made the changes for all descriptions (action and parameter)